src/elements/content-preview/ContentPreview.js

```diff
@@ -1113,11 +1123,18 @@ class ContentPreview extends React.PureComponent<Props, State> {
      */
     onVersionChange = (version?: BoxItemVersion, additionalVersionInfo: AdditionalVersionInfo = {}): void => {
         const { onVersionChange }: Props = this.props;
+        const { location } = additionalVersionInfo;
+
         this.updateVersionToCurrent = additionalVersionInfo.updateVersionToCurrent;
 
         onVersionChange(version, additionalVersionInfo);
+
         this.setState({
             selectedVersion: version,
+            startAt: {
+                unit: location.type === 'page' ? 'pages' : undefined,
```

Comment:
Will this easily extend to support other file/location types in the future?

Reply:
Not really, preview sdk only supports `pages` and `seconds` so I figured when we enhance this in the future we could address it then but I'll add a helper method to do the mapping to make it more extensible